sandbox/color_names.py

```diff
@@ -0,0 +1,46 @@
+import rich.repr
+from rich.align import Align
+from rich.console import RenderableType
+from rich.panel import Panel
+from rich.pretty import Pretty
+
+from textual._color_constants import COLOR_NAME_TO_RGB
+from textual.app import App, ComposeResult
+from textual.widget import Widget
+from textual.widgets import Placeholder
+
+
+@rich.repr.auto(angular=False)
+class ColorDisplay(Widget, can_focus=True):
```

Comment:
just a quick "sandbox" app I added, just because I wanted to pick a colour for my new "vertical" sandbox so I wanted to be able to visualise our available colours :smile: 

https://user-images.githubusercontent.com/722388/167121946-7ade143f-672c-4fda-b066-b8c18e385284.mp4